modules/support-gather-data.adoc

```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
 ====
 If this command fails, for example if you cannot schedule a Pod on your cluster, then use the `oc adm inspect` command to gather information for particular resources. Contact Red Hat Support for the recommended resources to gather.
 ====
++
+[NOTE]
+====
+If your cluster is using a restricted network you must import the default must-gather image, or mirror the latest must-gather image, and then run the `oc adm must-gather` command.
```

Comment:
If your cluster is in a restricted network, you probably don't have internet access to run this command. You probably need to actually mirror this content onto the machine that hosts the registry.